Contents/Code/__init__.py

```diff
@@ -61,7 +61,7 @@ def search(self, results, media, lang, manual):
             return
 
         search_helper.media.artist = String.StripDiacritics(
-                search_helper.media.artist
```

Comment:
Please make a new commit, reverting the spacing changes here and below on 288-290

Reply:
Done!